tests/unit/amo/components/TestAddonMoreInfo.js

```diff
@@ -259,117 +255,116 @@ describe(__filename, () => {
         url: 'http://license.com/',
       },
     });
+    render({ addon });
 
-    const root = render({ addon });
-    const link = root.find('.AddonMoreInfo-license-link');
-
-    expect(root.find('.AddonMoreInfo-license')).toHaveProp('term', 'License');
-    expect(link.children()).toIncludeText(licenseName);
-    expect(link).toHaveProp('to', `/addon/${addon.slug}/license/`);
+    expect(screen.getByText('License')).toBeInTheDocument();
+    const link = screen.getByText(licenseName);
+    expect(link).toHaveAttribute(
+      'href',
+      `/en-US/android/addon/${addon.slug}/license/`,
```

Comment:
We need to add `/en-US/android/` to the beginning of the link in this assertion as that's what gets rendered. The previous test was just checking the `to` prop of the `Link` component, so it didn't need to add those to the assertion.

We tend to be very explicit in addons-frontend tests, so if we want to continue with that practice we should probably set up the data in the test which sets a `lang` and `app`, and then use those values in this assertion. I think, though, as we are rewriting these tests, that we don't really need to do that. I think it's fine to just hardcode `/en-US/android/` here because we know that will be the value based on the defaults for the test. I think this is consistent with how a lot of tests are written for addons-server, where, rather than explicitly setting up all the data in the test, the tests rely on existing fixtures and just assert hardcoded values in the test. As this isn't meant to be a test of the `Link` component, I think it's fine to do what I have done here, but I did want to call out that it's not normally what we would do in an addons-frontend test, so we can discuss if we are okay with moving in this direction.